src/classes/Commands/Commands.ts

```diff
@@ -783,16 +832,25 @@ export default class Commands {
                 this.weaponsAsCurrency.enable ? this.bot.craftWeapons : [],
                 this.weaponsAsCurrency.enable && this.weaponsAsCurrency.withUncraft ? this.bot.uncraftWeapons : []
             );
+
+        if (amount > 0) {
+            const cartAmount = cart.getTheirCount(params.sku);
```

Comment:
Why is there a possibility to have 0 amount. Shouldn't that be caught at the start and returned an error.

Reply:
Because you can use a value less than 0 to remove things you've added to your cart.

https://github.com/TF2Autobot/tf2autobot/blob/a1497b54d2630b40ecbcd8cef7b8a8a4ab2d6400/src/classes/Commands/Commands.ts#L854